src/test/components/PermissionsChecker.test.tsx

```
@@ -0,0 +1,220 @@
+      ...mockKesselContext,
+      isLoading: kesselLoading,
+    };
+
+    return render(
+      <Provider store={store}>
+        <KesselRbacAccessContext.Provider value={contextValue}>
+          <PermissionsChecker>
+            <Children />
+          </PermissionsChecker>
+        </KesselRbacAccessContext.Provider>
+      </Provider>
```

ESLint

[error] Insert `,`

(prettier/prettier)